src/OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -406,8 +406,9 @@ void LiveFunctionsDataView::OnDataChanged() {
   const absl::flat_hash_map<uint64_t, orbit_grpc_protos::InstrumentedFunction>&
       instrumented_functions = app_->GetCaptureData().instrumented_functions();
   for (const auto& pair : instrumented_functions) {
```

Comment:
pers pref: while you are at it
```suggestion
  for (const auto& [unused, instrumented_function] : instrumented_functions) {
```

Reply:
done